Makefile

```diff
@@ -53,6 +53,15 @@ test-coverage: test
 test-coverage-clean:
 	git ls-files --others --ignored --exclude-standard | grep "coverprofile$$" | xargs rm
 
+generate-tools:
+ifeq (, $(shell which gojsonschema))
+	go get github.com/atombender/go-jsonschema/cmd/gojsonschema
+endif
+
+generate-messages: generate-tools 
+	mkdir -p internal/ansible/model/message/
+	gojsonschema --yaml-extension yaml -p message internal/ansible/schema/ansibleRunnerJobEvent.yaml > internal/ansible/model/message/runner-job-event-gen.go 
```

Comment:
Why not use `-o` instead of stoud out redirect `>`?

Reply:
You're right! Thanks!